applications/nrf_desktop/src/modules/usb_state.c

```diff
@@ -37,13 +37,28 @@ LOG_MODULE_REGISTER(MODULE, CONFIG_DESKTOP_USB_STATE_LOG_LEVEL);
 #define CONFIG_USB_HID_PROTOCOL_CODE -1
 #endif
 
+#define USB_HID_SELECTIVE_SUB	IS_ENABLED(CONFIG_DESKTOP_USB_SELECTIVE_HID_SUBSCRIPTION)
+
+#if USB_HID_SELECTIVE_SUB
+  #include "usb_state_def.h"
+#else
+  static const uint16_t usb_hid_selected_report_bitmask[0];
```

Comment:
there is no point in creating this variable beside appeasing the compiler
let's simply control checking code with pragma

Reply:
Default value will be provided with util_listify macro.